mmv1/products/bigquerydatapolicyv2/DataPolicy.yaml

```diff
@@ -0,0 +1,179 @@
+# Copyright 2024 Google Inc.
```

Comment:
nit: 2025, here and in the other YAML. Looks like the Autogen tool needs to update this...